spec/classes/mod/fcgid_spec.rb

```diff
@@ -102,8 +99,8 @@
     let :facts do
       {
         :osfamily                  => 'FreeBSD',
-        :operatingsystemrelease    => '10',
-        :operatingsystemmajrelease => '10',
+        :operatingsystemrelease    => '9',
+        :operatingsystemmajrelease => '9',
```

Comment:
From the revert of your commit that didn't explain the reason of this change.

I can apply it again, once I know what you canted to fix.